src/packages/shared-utils/package-check.ts

```diff
@@ -28,10 +28,12 @@ export const PackageCheck = ({
   raiWithdrawnDate,
   raiWithdrawEnabled,
   authority,
+  appkParentId,
 }: opensearch.main.Document) => {
   const planChecks = {
     isSpa: checkAuthority(authority, [Authority.MED_SPA, Authority.CHIP_SPA]),
     isWaiver: checkAuthority(authority, [Authority["1915b"]]),
+    isAppk: checkAuthority(authority, [Authority["1915c"]]) && !appkParentId,
```

Comment:
@pkim-gswell Can you get a test case written for this attribute to maintain coverage on this utility?